rayon-core/build.rs

```diff
@@ -0,0 +1,5 @@
+// We need a build script to use `link = "rayon-core"`.  But we're not
+// *actually* linking to anything, just making sure that we're the only
+// rayon-core in use.
+fn main() {
+}
```

Comment:
The empty build script should still declare dependencies. The default is that the build script (and thus the whole crate) is recompiled if any file changes. Including for example a README in the same directory.